dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -87,9 +87,9 @@ BEGIN
     EDITTEXT IDC_ADVAPPEARANCE_SIZE_E, 142, 170, 38, 13, ES_RIGHT | WS_GROUP
     CONTROL "", IDC_ADVAPPEARANCE_SIZE_UD, UPDOWN_CLASS, UDS_AUTOBUDDY | WS_BORDER | WS_GROUP |
             UDS_ALIGNRIGHT | UDS_ARROWKEYS | UDS_SETBUDDYINT | WS_CHILD | WS_VISIBLE, 172, 170, 10, 13
-    LTEXT "Couleur 1 :", IDC_ADVAPPEARANCE_COLOR1_T, 180, 160, 34, 9
+    LTEXT "Couleur 1 :", IDC_ADVAPPEARANCE_COLOR1_T, 180, 160, 34, 9//FIXME: add back accelerator which was destroyed during 0.4.15-dev'ing. I used &1 for the backport
     CHECKBOX "-", IDC_ADVAPPEARANCE_COLOR1_B, 184, 170, 28, 13, BS_PUSHLIKE | BS_BITMAP
-    LTEXT "Couleur 2 :", IDC_ADVAPPEARANCE_COLOR2_T, 215, 160, 34, 9
+    LTEXT "Couleur 2 :", IDC_ADVAPPEARANCE_COLOR2_T, 215, 160, 34, 9//FIXME: add back accelerator which was destroyed during 0.4.15-dev'ing. I used &2 for the backport
```

Comment:
Most of our historic translations did place the 2 accels here as &1 and &2. Which is a good choice here, if one doesn't know what Windows does, because it has no risk of collision-generation. I can apply for french whatever you want to see appled. But not based on a Windows 7 screenshot, but only based on a 2k3sp2 screenshot. If you cannot provide that, I will keep the FIXME comment there instead.

Reply:
I followed the upper one of those 2 screenshots entirely. It has much better accels than the lower one, as it avoids placing any on the small L: &l . It is also the 2k3sp2 one.

Reply:
One of my points with these two screenshots was also to emphasize that since Windows, for the same dialog between different versions, change their accelerators and this doesn't cause any problems, this means that we don't have to blindly follow Windows also for copying 100% the same sentences and putting the accelerators in the same places, when it happen it would be more logical to place them elsewhere.

Reply:
I am aware that MS changes the accels between different Windows versions. And I am aware that they are often better in 2k3sp2, than they are on Win Vista+. Your 2 screenshots here actually one good example. Your black screenshot obviously has worse accels than the grey one. That is because the black one is a Vista+ dlg. I am aware that we do not to 100% have to follow MS with the accels. But imho it is the wisest thing we can do to 1.) have accels that already passed the test of time 2.) some people might be familiar with them. 3.) we do have a clear definition-of-done then for a specific target.
Ftr: Back in the days where real humans did those accels at MS (read: XP/2k3 era) the end-result was often better.

Reply:
> 1.) have accels that already passed the test of time

They certainly haven't passed the test of time, since Windows XP/2003 is from 2001-2003-ish, while Vista is from 2007 (and some accelerators changed, others haven't, etc etc.), and now we are in 2024.

Reply:
Tell me one accelerator that is better in the Vista screenshot.
I can tell you one that is worse, and that is the "Cou&ler 1 :", it violates MS own principles btw.